### DISCUSSION TITLE
Version number in filename

### DISCUSSION THREAD

**Discussion by gedger** [aidanmacgregor/EE_WiFi-BT_WiFi-Autologin-OpenWRT#6] [Announcements]
Can I suggest removing the version from the filename as it's a bit confusing having V5 inside of V4. Plus it will make upgrading easier. Perhaps also moving to all lower case for the openwrt scripts, the Linux way.....
Thanks

Richard

**Comment by aidanmacgregor** [comment]
Fair point, sounds like a good idea :)

**Comment by aidanmacgregor** [comment]
done!
